src/box/sql/build.c

```diff
@@ -1125,34 +1155,49 @@ vdbe_emit_ck_constraint_create(struct Parse *parser,
 	struct Vdbe *v = sqlGetVdbe(parser);
 	assert(v != NULL);
 	/*
-	 * Occupy registers for 5 fields: each member in
-	 * _ck_constraint space plus one for final msgpack tuple.
+	 * Occupy registers for 20 fields: each member in _func space plus one
+	 * for final msgpack tuple.
 	 */
-	int ck_constraint_reg = sqlGetTempRange(parser, 7);
-	sqlVdbeAddOp2(v, OP_SCopy, reg_space_id, ck_constraint_reg);
-	sqlVdbeAddOp4(v, OP_String8, 0, ck_constraint_reg + 1, 0,
+	int regs = sqlGetTempRange(parser, 20);
```

Comment:
Given that the storage format has changed, and for fkeys too, is there a plan to update the upgrade.lua script to make the existing users' schemas up to date?

Reply:
Added.